versioned_docs/version-0.4/troubleshooting.md

```diff
@@ -223,4 +212,9 @@ When you see an error like the one below ...
 Error opening a gzip reader for /tmp/getter154967024/archive: gzip: invalid header
 ```
 
-... the content of the helm chart is incorrect. Manually download the chart to your local machine and check the content.
\ No newline at end of file
+... the content of the helm chart is incorrect. Manually download the chart to your local machine and check the content.
+
+### Migrate the local cluster to the Fleet default cluster workspace?
```

Comment:
Do we want to replicate this section for other versions too, to make it explicit that local cluster migration is not supported?